carbonj.service/src/main/java/com/demandware/carbonj/service/db/points/StagingFileSetCollector.java

```diff
@@ -19,34 +20,34 @@
 public class StagingFileSetCollector
 {
     private static final Logger log = LoggerFactory.getLogger( StagingFileSetCollector.class );
-    private File dir;
+    private final File dir;
 
     public StagingFileSetCollector(File dir)
     {
         this.dir = Preconditions.checkNotNull( dir );
     }
 
-    synchronized public List<SortedStagingFile> collectEligibleFiles(Map<StagingFileSet, StagingFile> files)
+    synchronized public List<SortedStagingFile> collectEligibleFiles(Map<StagingFileSet, StagingFile> files, String dbName)
     {
-        List<SortedStagingFile> sortedFiles = new ArrayList<>(  );
-        List<StagingFileSet> names = new ArrayList<>( files.keySet() );
-        names.sort( (o1, o2) -> o1.from - o2.from );
+        List<SortedStagingFile> sortedFiles = new ArrayList<>();
+        List<StagingFileSet> names = new ArrayList<>( files.keySet().stream().filter(fs -> fs.dbName.equals(dbName)).toList() );
+        names.sort(Comparator.comparingInt(o -> o.from));
 
         names.stream()
                 .filter( fs -> fs.needsCollection( files.get( fs ).lastModified() ) )
                 .forEach( fs ->
                                 {
-                                    log.debug( "processing staging file: [" + fs + "]" );
+                                    log.info( "processing staging file: [" + fs + "]" );
                                     StagingFile f = files.remove( fs );
                                     f.close();
                                     Optional<String> lastSorted = fs.getLastSortedFileName( dir );
-                                    log.debug( "sorting ..." );
-                                    SortedStagingFile sortedFile = f.sort(lastSorted);
-                                    log.debug("sorted file: [" + sortedFile + "]");
+                                    log.info( "sorting ..." );
+                                    SortedStagingFile sortedFile = f.sort(lastSorted, dbName);
+                                    log.info("sorted file: [" + sortedFile + "]");
                                     sortedFiles.add( sortedFile );
                                 }
                 );
-        log.debug("sorted files: [" + sortedFiles + "]");
+        log.info("sorted files: [" + sortedFiles + "]");
```

Comment:
Change these back to debug. Logs will be very verbose.

Reply:
done.